ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -94,6 +94,31 @@ spec:
             fsType: xfs
             devicePaths:
               - /dev/sdb2
+    # manually create PVs if they are not dynamically created by local-storage-operator
+    - fileName: StoragePV.yaml
+      policyName: "config-policy"
+      metadata:
+        name: pv-storage-ptp
+      spec:
+        storageClassName: "example-storage-class-1"
+        capacity:
+          storage: 10Mi
+        local:
+          # manually create ptp directory if local-storage is shared with bmer
+          # path: /mnt/local-storage/ptp
+          path: /mnt/local-storage
+    - fileName: StoragePV.yaml
+      policyName: "config-policy"
+      metadata:
+        name: pv-storage-bmer
+      spec:
+        storageClassName: "example-storage-class-1"
+        capacity:
+          storage: 10Mi
+        local:
+          # manually create bmer directory if local-storage is shared with ptp
```

Comment:
We need to make sure the examples provided works out of box without manual intervention.

Reply:
updated